hack/e2e-kind-config.yaml

```diff
@@ -1,13 +1,13 @@
 # this config file contains all config fields with comments
 kind: Cluster
 apiVersion: kind.x-k8s.io/v1alpha4
-# 1 control plane node and 3 workers
+# 1 control plane node and 1 workers
```

Comment:
travis runs with 1 CP node and 3 worker nodes, so local will run only with 1 CP and 1 worker?

Reply:
@asm582, 
I believe that all e3e tests run on 1 control plane + 1 node. AFAIK there's no way to add more nodes to a running cluster.  I was merely updating the coment to reflect what was in the configuration file.